api/routes/device_test.go

```diff
@@ -85,21 +92,27 @@ func TestDeleteDevice(t *testing.T) {
 		expectedStatus int
 	}{
 		{
-			title: "returns Ok when deleting an existing device",
-			uid:   "123",
-			requiredMocks: func() {
-				mock.On("DeleteDevice", gomock.Anything, models.UID("123"), "").Return(nil)
-			},
-			expectedStatus: http.StatusOK,
+			title:          "fails when bind fails to validate uid",
+			uid:            "",
+			requiredMocks:  func() {},
+			expectedStatus: http.StatusNotFound,
```

Comment:
in cases where the Status is the only expected output, it was kept the 'ExpectedStatus' parameter instead of the Expected structure in order to maintain a better standard in the codes